packages/components/ContextualMenu/src/SubmenuItem.types.ts

```diff
@@ -14,7 +15,7 @@ export type SubmenuItemProps = ContextualMenuItemProps;
 export type SubmenuItemState = ContextualMenuItemState;
 
 export interface SubmenuItemSlotProps {
-  root: React.PropsWithRef<IViewProps>;
+  root: React.PropsWithRef<PressablePropsExtended>;
```

Comment:
You might need to fork this based on platform, too

Reply:
The file SubmenuItem.types.macos.ts I added should take care of that, doesn't it?